src/madhavi_Khasbage/ArrayPrograms/FingMiddleCharOfStr.java

```diff
@@ -0,0 +1,33 @@
+package madhavi_Khasbage.ArrayPrograms;
+
+/*Create a method which will return middle character of each String. Print output in main method.
+String[] arr = {"Techno", "Hello", "Credits"};
+output : c l d
+*/
+
+public class FingMiddleCharOfStr {
+
+	String getMidCharString(String[] arrStr) {
+		String midChar = "";
+		char ch;
+		int indx1;
+		for (int index = 0; index < arrStr.length; index++) {
+			int strLength = arrStr[index].length();
```

Comment:
take reference of below code

https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1109/files